Week-2/portfolio-exercise/index.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
```

Comment:
1. Add margin between 2 article columns for desktop.
2. For single column layout, make article text center aligned
3. For single column layout, make footer text center aligned
4. Margin between footer text and the "article.text-center"
5. Margin between labels and inputs.
6. Make input form buttons center aligned for single column.
7. Make radio buttons center aligned for single column.

Reply:
fixed